infrastructure/images/k6-action/default_scenarios/soak.json

```
@@ -0,0 +1,8 @@
+{
+    "stages": [
+        {
+            "duration": "480m",
```

Nit, 8h is easier to read than 480m

```suggestion
            "duration": "8h",
```